whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -215,7 +212,12 @@ private RdapObject getRdapObject(final String requestUrl,
             rdapResponse.getRemarks().add(createRemark(rpslObject));
         }
 
-        rdapResponse.getEvents().add(createEvent(lastChangedTimestamp));
+        if (!rpslObject.containsAttribute(AttributeType.CREATED) || !rpslObject.containsAttribute(AttributeType.LAST_MODIFIED)){
```

Comment:
No need for this block, `getValueForAttribute` will throw an exception if it can't find the attribute